js/src/Ice/OutgoingConnectionFactory.js

```diff
@@ -618,20 +619,16 @@ class ConnectCallback {
     }
 
     connectionStartFailedImpl(ex) {
-        if (ex instanceof LocalException) {
```

Comment:
I removed this check because I don't understand it. What other kind of exception do we get here?

Note that handleConnectionException really means "log connection exception".